src/interfaces/common.interfaces.ts

```diff
@@ -399,6 +399,7 @@ export interface VehicleOdometer {
 export interface VehicleStatusOptions {
   refresh: boolean;
   parsed: boolean;
+  useInfo?: boolean;
```

Comment:
This is really only used for CA region from what I could tell and I guess ok for now.